Tasks/AzureFileCopy/README.md

```diff
@@ -82,6 +82,8 @@ The parameters of the task are described in details, including examples, to show
 
  * **Blob Prefix**: A prefix for the Blobs that can be used to filter the blobs like appending the Build number to the blobs, so that all the blobs with the same build number can be downloaded from the Container.
 
+* **Additional Arguments**: Additional [AzCopy.exe](https://azure.microsoft.com/en-us/documentation/articles/storage-use-azcopy/) arguments that will be applied when uploading to blob or uploading to VM like, /MT. We use "/S", "/Y" and "/Z" as default parameters.
```

Comment:
Should you call out explicitly that the default parameters cannot be overridden.